api/logger.go

```diff
@@ -23,7 +24,11 @@ func registerBoletoLogger(c *gin.Context) {
 
 	l := loadBankLog(c)
 
-	l.RequestApplication(boleto, c.Request.URL.RequestURI(), util.HeaderToMap(c.Request.Header))
+	if config.Get().MockMode {
```

Comment:
Opa, uma outra abordagem que talvez o código fique mais simples, seria a responsabilidade de usar o mockMode no **maskAuthenticationNode**, por exemplo abaixo 😀 
```suggestion
	func (a Authentication) maskAuthenticationNode() Authentication {
	if config.Get().MockMode {
		return a
	}
	
	authentication := Authentication{}
	if isNotEmptyString(a.Username) {
		authentication.Username = "*****"
	}
	if isNotEmptyString(a.Password) {
		authentication.Password = "*****"
	}
	if isNotEmptyString(a.AuthorizationToken) {
		authentication.AuthorizationToken = "*****"
	}
	if isNotEmptyString(a.AccessKey) {
		authentication.AccessKey = "*****"
	}
	return authentication
}

```